sys/net/application_layer/Kconfig.coap

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config HAS_PROTOCOL_COAP
+    bool
+    help
+        Specifies that CoAP support is present.
+
+menuconfig KCONFIG_COAP
+    bool "Configure CoAP generic options"
+    depends on HAS_PROTOCOL_COAP
+    help
+        Configure CoAP generic options via Kconfig.
+
+if KCONFIG_COAP
+
+config COAP_ACK_TIMEOUT
+    int "Timeout in seconds for a response to a confirmable request"
+    default 2
+    help
+        This value is for the response to the initial confirmable message. The
+        timeout doubles for subsequent retries. To avoid synchronization of
+        resends across hosts, the actual timeout is chosen randomly between
+        @ref CONFIG_COAP_ACK_TIMEOUT and
+        (@ref CONFIG_COAP_ACK_TIMEOUT * @ref CONFIG_COAP_RANDOM_FACTOR_1000 / 1000).
```

Comment:
The doxygen `@ref` do not work with kconfig, maybe simply omit them in the help here.

Reply:
I know Kconfig files are not generating documentation yet, but I plan to do so in the future. So this is just so we don't lose track of references that are currently on the docs.

Reply:
Is there a way to add the marker so that it doesn't show up in the kconfig help?

Reply:
See #13916

Reply:
Leaving a note that this is being addressed in #13916

Reply:
> See #13916

Didn't see that :smile: